emulator.c

```diff
@@ -112,8 +168,17 @@ execute_instruction(i8080 *cpu, uint8_t opcode)
       printf("ANA A");
       break;
     case 0xaf: // NOLINT
-      printf("XRA A");
-      break;
+      {        // XRA A
+        uint8_t result = cpu->a ^ cpu->a;
+        update_sign_flag(cpu, result);
+        update_zero_flag(cpu, result);
+        update_parity_flag(cpu, result);
+        update_carry_flag(cpu, false);
+        update_aux_carry_flag(cpu, result, 0xFF); // NOLINT
```

Comment:
Is this correct for aux carry?

This opcode just sets A to 0. I'm having a bit of trouble wrapping my head around aux carry's behavior here. I think this should clear the aux carry flag? I couldn't find it in the 8080 manual.

https://retrocomputing.stackexchange.com/questions/14977/auxiliary-carry-and-the-intel-8080s-logical-instructions

Reply:
This is what I have from section 4-9 (page 53) in the manual I have (think it's the one I put in the resources directory. ![image](https://user-images.githubusercontent.com/122326493/236702093-f7138840-7669-46f4-8ff7-a6d9c34137c5.png)

Reply:
Great - thanks for the citation - had trouble finding it for some reason.